### DISCUSSION TITLE
Additional bookmark files

### DISCUSSION THREAD

**Discussion by smallwat3r** [qutebrowser/qutebrowser#7507] [Q&A] [answered]
We can add bookmarks under `.qutebrowser/bookmarks/urls`. 

But is there any way to use additional files such as: `.qutebrowser/bookmarks/private-urls`?

I store my dotfiles online in a public git repository, and I would like to keep some urls public, but keep some other private (work related urls for example).

An option would be to leverage the quickmarks feature for this, as these urls are stored in a separate file (`.qutebrowser/quickmarks`), but I would prefer to use the bookmarks option if that's possible.

**Comment by The-Compiler** [comment, marked as answer]
This isn't possible currently. Originally, it was intended to use the filesystem structure for bookmark subfolders, but that was never actually implemented. It's probably not going to be either, as we decided to move to a more flexible JSON-based format with #882. After that's implemented, though, it *might* make sense to allow reading from multiple files. I opened #7512 to track this.
